docs/DeveloperGuide.md

```diff
@@ -232,17 +232,18 @@ The activity diagram shows the general sequence of steps when a user interacts w
 
 ### Edit person feature
 
-#### Implementation
-
 #### Design considerations
+**Aspect: Check if the person with person id exists before editing**
+- This is to ensure no unwanted errors occur while editing the person.
 
 <br>
 
 ### Delete person feature
 
-#### Implementation
-
 #### Design considerations
+**Aspect: deleting a person should also remove appointments linked to the person**
+- Alternative 1 (Current choice): Deleting a person will also remove any appointments with the personId of that person, this is to prevent any appointments with person ids that are non-existent.
+- Alternative 2: We assume the user deletes the appointment, however this is not a good assumption since the user might forget doing so.
```

Comment:
Would it be possible to rephrase it to keep it consistent with the formatting? Perhaps it can be rephrased as 

**Aspect: Deleting a person should also remove appointments linked to the person**
- **Alternative 1 (current choice):** Deleting a person will also remove any appointments with the `personId` of that person.
  - Pros: This prevents the case where appointments are linked to `personIds` that are non-existent.
- **Alternative 2:** Deleting a person will not remove any appointments with the `personId` of that person.
  - Cons: This assumes the user would delete the appointments linked to the deleted person's `personId`. However, the user might forget to do so.